packages/react-ui/src/app/router/index.tsx

```diff
@@ -77,6 +78,12 @@ const routes = [
     path: '/embed',
     element: <EmbedPage></EmbedPage>,
   },
+  {
+    path: '/embed/connections',
+    element: (
+      <EmbeddedConnectionDialogWrapper></EmbeddedConnectionDialogWrapper>
```

Comment:
Nit: This should be without wrapper, as wrapper is relative to component implementation rather than what it actual does

Remember declarative is ftw